src/server/routes/status.js

```diff
@@ -0,0 +1,25 @@
+module.exports = (app, db, addEvent, log) => {
+  app.get('/status', (req, res) => {
+    db.all('SELECT DISTINCT host FROM events', (err, events) => {
```

Comment:
Get hosts from the yaml config instead, don't want to show hosts that have been removed from config but are still in db